main.go

```diff
@@ -220,9 +220,13 @@ func runCopy(
 			"targets":     inputConf.Schema + `."` + inputTable.Name + `"`,
 			"vacuum_mode": "delete",
 		}
-		// If we truncated, run an analyze as well
+		// If we truncated, run an analyze as well (threshold 0.001%). Otherwise,
+		// only analyze if we've changed enough rows (> 10%, redshift default)
+		cleanupArgs["analyze_mode"] = "full"
 		if truncate {
-			cleanupArgs["analyze_mode"] = "full"
+			cleanupArgs["analyze_threshold"] = "0.001"
+		} else {
+			cleanupArgs["analyze_threshold"] = "10"
```

Comment:
The 10% default got us into trouble in the past with hmlfvs because it was longer than 28 days of data.  We could make this smaller but we're also running it at 0.001% every night so maybe this isn't really necessary.

Reply:
Or we can use a smaller threshold like 1%, that won't be triggered on every run but would still run if our nightly analyze fails for some reason.

Reply:
I think we've got a lot of tables that could benefit from a smaller percentage. I like the idea of having a default that's a bit more optimized than 10%. I made it so small since I wanted to make sure we weren't at risk of running into the 30-day window for hmlfvs again any time soon.

Reply:
That said, I think this distinction might not be necessary. It we truncate the table, then we've changed 50% of the data, so any (reasonable) analyze should do it. It was just that it's important to reprocess, unlike historical tables, where there are probably fewer underlying changes.

Reply:
Agreed, I was thinking that too when adding this but I wanted it to be more explicitly written out. I will put it directly into the blob with a more detailed comment, should achieve the same thing.

Reply:
I'll use 1% as the threshold